### PR TITLE
Data acquisition chapter re-write

### PR DESCRIPTION
**Pull request by mariaruth**
Edited data acquisition chapter to follow the revised skeleton. Removed data quality assurance and edited intro accordingly. 

@kbjarkefur , a few points to note when you are reviewing:

- [ ] the flow with section 2 (high-quality data collection) still feels strange to me. i made a bunch of edits to try to make the three sections go together, but any thoughts on the overall chapter organization would be helpful.
- [ ] i added content and edited the section on data acquisition heavily, extra attention there would be good
- [x] the handling data securely section needs more content to be applicable for all types of data (it still reads as very survey-focused). i made some edits to this effect, but there are two places where we need additional content and I think you are better placed to add. could you draft something and share back? in both a paragraph should suffice. First, the 'storing data securely' section doesn't cover 'big' data, i.e. data that can't be stored in encrypted folder on desktop). Second, in the 'sharing data securely' section, we talk about de-identifying data but should also add a bit on other transformations for safe use, such as creating aggregates (as with the COVID mobility data).

### CODE REVIEW

File: chapters/data-collection.tex [hunk 1]
```diff
@@ -1,140 +1,97 @@
...
 
 \begin{fullwidth}
+High-quality data is essential to most modern development research.
+Many research questions require novel data, 
+because no source of publicly available data addresses the 
+inputs or outcomes of interest for the relevant population.
+Data acquisition can take many forms, including: 
+primary data generated through surveys; 
+private sector partnerships granting access to new data sources, such as administrative and sensor data;
+digitization of paper records, including administrative data;
```
Comment (kbjarkefur) [inline chapters/data-collection.tex:11]:
```suggestion
digitization of paper records, including administrative data; web scraping;
```
I think this should be a category of its own here.

File: chapters/data-collection.tex [hunk 2]
```diff
@@ -1,140 +1,97 @@
...
-and what terms-of-use you require. 
-
-As a researcher, it is your responsibility to respect the rights
-of people who own data and people who are described in it;
-but it is also your responsibility to make sure
-that information is as available and accessible it can be.
+Rather, they must enter into data licensing agreements 
+to access the data and publish derivative work. 
+These agreements should make clear from the outset whether the 
+research team can make the original data, or any portion thereof, public.
```
Comment (kbjarkefur) [inline chapters/data-collection.tex:46]:
```suggestion
research team can make the original data, any portion thereof, or derivatives\sidenote{
  \textbf{Derivatives} of a data can be indicators, aggregates etc. based on the data.} 
 of the data, public. 
```
Don't know if this is the best way to do it, but I think we should include derivatives already here.

**Comment by kbjarkefur** [comment]
https://github.com/worldbank/d4di/blob/04ea6e812ff9e3f57f138c982d6522175e92460e/chapters/data-collection.tex#L75-L82

The last two lines above. Can we really say that it must stipulate that the research teams own the data? What is important is that the research team _either_ owns the data or have full right to it? In GDPR countries one could argue that only the respondents can own the data, and in some places the gov pay for it and they owns it but research teams have legal right to use it. Or am I missing something?

**Comment by kbjarkefur** [comment]
https://github.com/worldbank/d4di/blob/04ea6e812ff9e3f57f138c982d6522175e92460e/chapters/data-collection.tex#L94-L106

This section is in the data owner ship section, but isnt it all about data licencing? I think this should be moved to next section, and there is some overlap in what is said already, so it can be shortened. For example:
https://github.com/worldbank/d4di/blob/04ea6e812ff9e3f57f138c982d6522175e92460e/chapters/data-collection.tex#L113-L115

File: chapters/data-collection.tex [hunk 3]
```diff
@@ -148,64 +105,52 @@ \subsection{Data licensing agreements}
...
-The metadata - documentation about the data - is critical for future use of the data.
-Metadata should include documentation of how the data was created, 
-what they measure, and how they are to be used.
-In the case of survey data, this includes the survey instrument and associated manuals; 
-the sampling protocols and field adherence to those protocols, and any sampling weights;
-what variable(s) uniquely identify the dataset(s), and how different datasets can be linked;
-and a description of field procedures and quality controls. 
-We use as a standard the Data Documentation Initiative (DDI), which is supported by the 
-World Bank's Microdata Catalog.\sidenote{\url{https://microdata.worldbank.org}}
+The World Bank has a template Data License Agreement which DIME follows.\sidenote{ \url{https://worldbankgroup.sharepoint.com/teams/ddh/SiteAssets/SitePages/ddh/DataLicenseAgreementTemplate_v4.pdf?cid=68a54269-bbff-4b47-846d-cab248ad7de1}}
```
Comment (kbjarkefur) [inline chapters/data-collection.tex:131]:
```suggestion
The World Bank has a template Data License Agreement which DIME follows.\sidenote{
\url{https://worldbankgroup.sharepoint.com/teams/ddh/SiteAssets/SitePages/ddh/DataLicenseAgreementTemplate_v4.pdf?cid=68a54269-bbff-4b47-846d-cab248ad7de1}}
```
just a line break so the line is not so wide. We cannot have line breaks before `\sidenote{` as it creates unwanted white space but right afterwards work.

Comment (kbjarkefur) [inline chapters/data-collection.tex:131]: This link is behind WB 2-factor authentication when I tested. Can you open this on a non-WB computer?

Comment (kbjarkefur) [reply chapters/data-collection.tex:131]: Actually, I couldnt even open on my WB computer. I get this message:

image

**Comment by kbjarkefur** [comment]
https://github.com/worldbank/d4di/blob/04ea6e812ff9e3f57f138c982d6522175e92460e/chapters/data-collection.tex#L136-L141

This is a bit too World Banky. Can we say the same thing without mentioning the World Bank? And I do not know how the term to `official use only` translates to any non-WB context. Maybe we should re-phrase this to the meaning of these WB terminology.

**Comment by kbjarkefur** [comment]
https://github.com/worldbank/d4di/blob/04ea6e812ff9e3f57f138c982d6522175e92460e/chapters/data-collection.tex#L476-L511

These two paragraphs are in the section "_handling data securely_" but this is about how to document data received from another organisation. This is a better fit int he "_Data acquisition_" section, but then there is almost nothing left here.

**Comment by kbjarkefur** [comment]
Overall I think this is a really good improvement to include new sources of data. I have a few comments above but all of them seems very solvable.

**Comment by kbjarkefur** [comment]
Responding to your three bullet points. 

1. I do not think it is weird that the section for primary data collection is longer than the section for acquiring secondary data from someone else. Primary data collection is more difficult and laborious than getting secondary data (as long as the counterpart is not being difficult when agreeing on data sharing agreement). But perhaps a sentence or two can be added justifying that.

2. I think the new part about data aquisition is good. The main thing missing are perhaps examples and other things that makes it more concrete and operational. I think the reader will have a good intuition for what is important, and an understanding of which questions are important, but not what to do next. I guess that is what is intended to go into the Wiki, is that right? I have some comments on the SharePoint link and the section that lists terminology specific to the WB.

3. Breaking up into sub-points:
    1. Agreed that the big data aspect is missing, but otherwise I do not think it is to survey data heavy. It is only the "_collecting data securely_" that is survey heavy, but that section is needed and it is not long. 
    1. I wonder if me too is not capable of writing a section about how to safeguard very large data stored in the cloud. Other sections of the book say consult a cybersecurity expert (for example [here](https://github.com/worldbank/d4di/blob/04ea6e812ff9e3f57f138c982d6522175e92460e/chapters/data-collection.tex#L455-L457)) and I think we have to say they something similar about cloud storage.
    1. I can add something about aggregation and other ways of making sensitive data less sensitive.

**Comment by mariaruth** [comment]
> https://github.com/worldbank/d4di/blob/04ea6e812ff9e3f57f138c982d6522175e92460e/chapters/data-collection.tex#L75-L82
> 
> The last two lines above. Can we really say that it must stipulate that the research teams own the data? What is important is that the research team _either_ owns the data or have full right to it? In GDPR countries one could argue that only the respondents can own the data, and in some places the gov pay for it and they owns it but research teams have legal right to use it. Or am I missing something?

- agree: addressed in 4d06c3eaea07fab10d6f6e7d41d188aaf52518dd

**Comment by mariaruth** [comment]
> https://github.com/worldbank/d4di/blob/04ea6e812ff9e3f57f138c982d6522175e92460e/chapters/data-collection.tex#L94-L106
> 
> This section is in the data owner ship section, but isnt it all about data licencing? I think this should be moved to next section, and there is some overlap in what is said already, so it can be shortened. For example:
> https://github.com/worldbank/d4di/blob/04ea6e812ff9e3f57f138c982d6522175e92460e/chapters/data-collection.tex#L113-L115

I added some language to clarify, because i did intend for this to be separate from the data licensing discussion -- it's meant to be thinking about data ownership downstream (beyond the research team use)

File: chapters/data-collection.tex (hunk 3 above)
Comment (mariaruth) [reply chapters/data-collection.tex:133]: oh bother. i thought i had found the external link. i now this is now publicly available, will dig to find a link that works.

**Comment by mariaruth** [comment]
> https://github.com/worldbank/d4di/blob/04ea6e812ff9e3f57f138c982d6522175e92460e/chapters/data-collection.tex#L136-L141
> 
> This is a bit too World Banky. Can we say the same thing without mentioning the World Bank? And I do not know how the term to `official use only` translates to any non-WB context. Maybe we should re-phrase this to the meaning of these WB terminology.

let's discuss. since it's referring to the template licensing agreement as a specific example, to me it makes sense that it refers to the Bank.

**Comment by mariaruth** [comment]
> https://github.com/worldbank/d4di/blob/04ea6e812ff9e3f57f138c982d6522175e92460e/chapters/data-collection.tex#L476-L511
> 
> These two paragraphs are in the section "_handling data securely_" but this is about how to document data received from another organisation. This is a better fit int he "_Data acquisition_" section, but then there is almost nothing left here. Not sure what to do.

Agree, i've moved this up to data acquisition
 2af71cfdf26185051fa3aee1654845cd1b0be8c4

**Comment by mariaruth** [comment]
> Responding to your three bullet points.
> 
> 1. I do not think it is weird that the section for primary data collection is longer than the section for acquiring secondary data from someone else. Primary data collection is more difficult and laborious than getting secondary data (as long as the counterpart is not being difficult when agreeing on data sharing agreement). But perhaps a sentence or two can be added justifying that.
good point, added a sentence 7bbcbfb4ca117652da85946857d65799c97fa31b

> 2. I think the new part about data aquisition is good. The main thing missing are perhaps examples and other things that makes it more concrete and operational. I think the reader will have a good intuition for what is important, and an understanding of which questions are important, but not what to do next. I guess that is what is intended to go into the Wiki, is that right? I have some comments on the SharePoint link and the section that lists terminology specific to the WB.
yes @Avnish95 it would be good to have examples on the wiki, perhaps templates for data licensing agreements from JPAL or IPA if they make them available 

> 3. Breaking up into sub-points:
>    
>    1. Agreed that the big data aspect is missing, but otherwise I do not think it is to survey data heavy. It is only the "_collecting data securely_" that is survey heavy, but that section is needed and it is not long. Since I am suggesting to move out the metadata section from _receiving data from development partners_ I could replace that with a section on what is good and bad methods to transfer files.
adding this would be great! and would round that last section out
>    2. I wonder if me too is not capable of writing a section about how to safeguard very large data stored in the cloud. Other sections of the book say consult a cybersecurity expert (for example [here](https://github.com/worldbank/d4di/blob/04ea6e812ff9e3f57f138c982d6522175e92460e/chapters/data-collection.tex#L455-L457)) and I think we have to say they something similar about cloud storage.
ok, could you add language to that effect? some general principles to keep in mind would be good, even if implementation will depend entirely on context and organizational preferences

>    3. I can add something about aggregation and other ways of making sensitive data less sensitive.
that would be super!

File: chapters/data-collection.tex [hunk 4]
```diff
@@ -194,18 +95,121 @@ \subsection{Receiving data from development partners}
...
+In others, it is owned by the people who collected it.
+In still more, it is highly unclear and there are varying norms.
+The best approach is always to consult with a local partner, 
+and enter into specific legal agreements establishing ownership, 
+access, and publication rights.
+This is particularly critical where confidential data is involved
+-- that is, when people are disclosing information to you
+that you could not obtain simply by observation or through public records. 
+
+For data generated for the research study, such as survey data,
```
Comment (kbjarkefur) [inline chapters/data-collection.tex:129]:
```suggestion
For primary data generated by the research team, such as survey data, 
```
This builds better on vocabulary we have already established

File: chapters/data-collection.tex [hunk 5]
```diff
@@ -194,18 +95,121 @@ \subsection{Receiving data from development partners}
...
+This is particularly critical where confidential data is involved
+-- that is, when people are disclosing information to you
+that you could not obtain simply by observation or through public records. 
+
+For data generated for the research study, such as survey data, 
+it is important to clarify up front who owns the data,
+and who will have access to the data.
+These details need to be shared with respondents when they are offered the opportunity 
+to consent to participate in the study. 
+If the research team is not collecting survey data directly --
```
Comment (kbjarkefur) [inline chapters/data-collection.tex:134]:
```suggestion
If the research team is not collecting the data directly --
```
Not specific to survey data

**Comment by kbjarkefur** [comment]
> > https://github.com/worldbank/d4di/blob/04ea6e812ff9e3f57f138c982d6522175e92460e/chapters/data-collection.tex#L94-L106
> > 
> > This section is in the data owner ship section, but isnt it all about data licencing? I think this should be moved to next section, and there is some overlap in what is said already, so it can be shortened. For example:
> > https://github.com/worldbank/d4di/blob/04ea6e812ff9e3f57f138c982d6522175e92460e/chapters/data-collection.tex#L113-L115
> 
> I added some language to clarify, because i did intend for this to be separate from the data licensing discussion -- it's meant to be thinking about data ownership downstream (beyond the research team use)

I have read this again. I think everything that is said is good, but I think an improvement that would have prevented my initial confusion is a re-ordering. I suggest this order of the _Data acquisition_ section:

* data licensing
* data ownership
* receiving data from dev partners

My initial confusion came from the fact the data ownership section discussed data licensing in the data ownership section. I think we need to talk licencing in the ownership section, but not as much the other way around, so it make sense to do licensing first. And since you deal with licensing and ownership before you receive the data, then I think that section should come last. Does that make sense?

File: chapters/data-collection.tex (hunk 3 above)
Comment (kbjarkefur) [reply chapters/data-collection.tex:189]: Did you find the public link?

**Pull request merged**